src/controllers/employee.js

```
@@ -1,180 +1,606 @@
-  for (let emp of departmentLeadId) {
-    if (emp.id && !ids.includes(emp.id)) {
-      ids.push(emp.id);
+const getTopFive = (results) => {
+  const arr = [];
+  const size = results.length > 6 ? 6 : results.length;
+  for (let i = 0; i < size; i++) {
+    const data = { id: i + 1 };
+    arr.push(data);
+    if (i < 5) {
+      data.label = results[i].label;
+      data.value = parseInt(results[i].value);
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)